bin/generate-rest.php

```diff
@@ -104,7 +104,7 @@
             $extractor = new Extractor($fileinfo);
 
             if ($pxml = $extractor->getFileContents('package2.xml')) {
-            } elseif ($pxml = $extractor->getFileContents('package.xml'));
+            } elseif ($pxml = $extractor->getFileContents('package.xml'))
```

Comment:
this is not a fix, rather a bug introduction (without this `;` the next line would be considered the body of the `elseif`)

whole this construct is a bit 'unorthodox'... maybe add empty block to state that it's intended that way